### PR TITLE
README.md: draft the first version introduction

### PR DESCRIPTION
**Pull request by kenplusplus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+it requires cloud native design to break monolithic inference pipeline in several
+microservices.
+
+![](docs/cnap_arch.png)
+
+The major microservices includes:
+
+| Microservice | Role | Description  |
+| ------------ | ---- | ----------- |
+| Transcoding Gateway | Data Source | Receive multiple real-time streams and do transcode |
```
Comment (leyao-daily) [inline README.md]: do transcoding or perform transcoding.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+| Frame Queue | Data Integration | Assign the input stream into specific work queue |
+| Infer Engine | Data Analytics | Infer the frame and send result to result broker |
+| Dashboard | Data Visualization | Rendering the result into client's single page application |
```
Comment (leyao-daily) [inline README.md]: Rendering -> Render, it should be in same format with the privous description.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Cloud Native AI Pipeline
+
+_(Disclaimer: This project is currently under active development, and as such, all source code may not be included in any release. This means that the code is subject to change without notice, and that any information contained within the code should be considered as work in progress.)_
+
+Cloud-native technologies can be applied to artificial intelligence (AI) for scalable
+application in dynamic environment such as public, private and hybrid cloud. But
```
Comment (leyao-daily) [inline README.md]: environments

File: README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+it requires cloud native design to break monolithic inference pipeline in several
```
Comment (leyao-daily) [inline README.md]: It can be more clear to change to "to **decomposes** monolithic inference pipeline **into**...". What do you think?

File: README.md
Comment (hwang37) [inline README.md:1]: all source code => all source codes

Comment (hwang37) [inline README.md:1]: artificial intelligence => Artificial Intelligence
(capital 1st character)

File: README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+it requires cloud native design to decomposes monolithic inference pipeline into several
+microservices.
+
+![](docs/cnap_arch.png)
+
+The major microservices includes:
+
+| Microservice | Role | Description  |
+| ------------ | ---- | ----------- |
+| Transcoding Gateway | Data Source | Receive multiple real-time streams and  perform transcoding |
```
Comment (leyao-daily) [inline README.md]: An extra space between `and` and `perform`

File: README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Cloud Native AI Pipeline
+
+_(Disclaimer: This project is currently under active development, and as such, all source code may not be included in any release. This means that the code is subject to change without notice, and that any information contained within the code should be considered as work in progress.)_
+
+Cloud-native technologies can be applied to artificial intelligence (AI) for scalable
+application in dynamic environments such as public, private and hybrid cloud. But
+it requires cloud native design to decomposes monolithic inference pipeline into several
```
Comment (leyao-daily) [inline README.md]: ops, sorry to make a typo in my privious review, change `decomposes` to `decompose`

File: README.md
Comment (kenplusplus) [reply README.md:1]: https://www.zhihu.com/question/41405648
source code (uncountable)
(computing, uncountable)
 Human-readable instructions in a programming language, to be transformed into machine instructions by a compiler, interpreter, assembler or other such system.

Comment (kenplusplus) [reply README.md:1]: good, fixed

File: docs/cnap_arch.png
Comment (hwang37) [inline docs/cnap_arch.png:1]: One comment:
Should we combine two "Infer Engine" boxes into one? It'll unify the flow style, from one box to 2nd, to 3rd, to 4th box.
Of cause, "Infer Engine" should have multiple layered boxes.

**Review by leyao-daily — APPROVED** [review]
LGTM. Thanks.

File: docs/cnap_arch.png
Comment (kenplusplus) [reply docs/cnap_arch.png:1]: Different infer engine use different inference model and mapping to different working queue. It is purposed for multiple infer engine BOX. This is "map-reduce" design.

**Comment by kenplusplus** [comment]
this is for #4

**Comment by leyao-daily** [comment]
Closes #4

**Pull request merged**